platform/mbed_lib.json

```diff
@@ -128,10 +128,6 @@
             "help": "Use the MPU if available to fault execution from RAM and writes to ROM. Can be disabled to reduce image size.",
             "value": true
         },
-        "minimal-printf-console-output": {
```

Comment:
Should `minimal-printf-enable-file-stream` also be removed given `mbed_minimal_putchar` no longer considers it?
If it should be removed, references in `mbed_printf.c/h` and `mbed_printf_wrapper.c` also should be removed.

Reply:
1f82adb